Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.14-alpine
+FROM golang:1.17-alpine@sha256:b35984144ec2c2dfd6200e112a9b8ecec4a8fd9eff0babaff330f1f82f14cb2a
```

Comment:
why the hash?

Reply:
Document on why to pin dependency https://github.com/ossf/scorecard/blob/main/docs/checks.md#pinned-dependencies